### ISSUE TITLE
Translate documentation into Spanish 

### ISSUE THREAD

**Issue by LaurenceM10** [tokio-rs/website#311]
It would be great to have documentation in other languages, such as Spanish for example.

**Comment by Lonami** [comment]
Maintaining two separate translations without any coordination is a bit tough. Furthermore, I think the documentation should be complete before any translation, since any changes would mean double the work, and finding people who can translate it.

**Comment by LaurenceM10** [comment]
@Lonami You're right. However having the documentation in several languages would make more developers start to try tokio.

The problem of many developers in my country is not to understand English, therefore they do not venture to try new languages or frameworks.

**Comment by carllerche** [comment]
I would love to have more translations of the documentation. Of course, this would require contributors to take on maintaining the documentation.

I don't think the docs will ever be "done", that said, there will be a large change once Tokio switches to `async / await`.

**Comment by Darksonn** [comment]
I will close this for now. If someone is actually interested in translating our guides, feel free to open a new issue.
